nyaa/api_torznab.py

```diff
@@ -101,71 +112,113 @@ def torznab_caps():
     return render_torznab_caps()
 
 
-def torznab_search():
-    page_size = app.config['RESULTS_PER_PAGE']
+def get_categories_filter(cat):
+    categories = []
+    if cat:
+        for item in cat.split(','):
+            # Parse 1_1 format for compatibility
```

Comment:
The code is functionally identical.  If cat is not falsey, then it gets split and assigned to items.  If it is falsey, then items becomes an empty list.  With an empty list `for item in items:` short circuits and skips the loop (same thing that `if cat:` accomplished).